.github/workflows/ci.yml

```diff
@@ -28,6 +28,12 @@ jobs:
       run: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile=minimal
     - name: Install Rust channel
       run: rustup toolchain install ${{matrix.rust}} && rustup default ${{matrix.rust}}
+    - name: Install `cargo wasi`
+      run: cargo install cargo-wasi
+    - name: Install `wasmtime`
+      run: curl https://wasmtime.dev/install.sh -sSf | bash
```

Comment:
CI seems to be failing. Could you also move this out into a new, dedicated job, instead of making every matrix job run the wasm tests?

Reply:
oh.
addition `source ~/.bashrc`.
cargo-wasi bin support only linux.